staging/src/k8s.io/bootstrap/OWNERS

```diff
@@ -0,0 +1,15 @@
+approvers:
+- caesarxuchao
```

Comment:
I don't really know this code area, please remove me from approvers and reviewers. Thanks.